### PR TITLE
Updated webview title with contents

### PR DESCRIPTION
**Pull request by tjcouch-sil**
image

- - -
This change is [Reviewable](https://reviewable.io/reviews/paranext/paranext-extension-text-collection/12)

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -26,8 +26,8 @@ const reactWebViewProvider: IWebViewProvider = {
         `${reactWebViewType} provider received request to provide a ${savedWebView.webViewType} web view`,
       );
     return {
-      ...savedWebView,
       title: 'Text Collection',
+      ...savedWebView,
```
Comment (tombogle) [inline src/main.ts:30]: What is the significance of changing the order of these?

File: src/main.ts
Comment (tombogle) [inline src/main.ts:1]: What is the significance of the dataProvider thing on line 10? I don't see that it is "used" anywhere.

**Review by tjcouch-sil — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/paranext/paranext-extension-text-collection/12)* status: 0 of 2 files reviewed, 2 unresolved discussions (waiting on @tombogle)

___
*[`src/main.ts` line 0 at r1](https://reviewable.io/reviews/paranext/paranext-extension-text-collection/12#gh-1367331160:-NhDDrkM58oLEOyT58jt:b-yya2fh) ([raw file](https://github.com/paranext/paranext-extension-text-collection/blob/466d637f8b719cccd85c01f1adf22608006e7142/src/main.ts#L0)):*
<details><summary><i>Previously, tombogle (Tom Bogle) wrote…</i></summary><blockquote>

What is the significance of the dataProvider thing on line 10? I don't see that it is "used" anywhere.
</blockquote></details>

Looks like code that is unused but has been there a while. I'd prefer not to remove it unless I need to make other changes.
___
*[`src/main.ts` line 30 at r1](https://reviewable.io/reviews/paranext/paranext-extension-text-collection/12#gh-1367325006:-NhDDyZk5PF5timpb8Xr:brgxslh) ([raw file](https://github.com/paranext/paranext-extension-text-collection/blob/466d637f8b719cccd85c01f1adf22608006e7142/src/main.ts#L30)):*
<details><summary><i>Previously, tombogle (Tom Bogle) wrote…</i></summary><blockquote>

What is the significance of changing the order of these?
</blockquote></details>

Previously, the `savedWebView.title` would be overwritten by the `title: 'Text Collection'` property. When I put `...savedWebView` below `title: 'Text Collection'`, that essentially makes `'Text Collection'` the default property value if `title` isn't in `savedWebView`, but it leaves `savedWebView`'s title alone if it is specified. So basically, this change made it so the web view title stays whatever it is when it is changed in the webview code. Otherwise, the title would reset to "Text Collection" every time it was loaded (then it would almost immediately but not quite immediately change to the correct title).

**Review by katherinejensen00 — APPROVED** [review]
:lgtm:

> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/paranext/paranext-extension-text-collection/12#-:-NhDEcg895ygDt7JdD3r:bnfp4nl)* status: all files reviewed, 2 unresolved discussions (waiting on @tombogle)

**Comment by tombogle** [comment]
*[`src/main.ts` line 0 at r1](https://reviewable.io/reviews/paranext/paranext-extension-text-collection/12#gh-1367331160:-NhEzhCsAqZykLXtti3n:b15jxlw) ([raw file](https://github.com/paranext/paranext-extension-text-collection/blob/466d637f8b719cccd85c01f1adf22608006e7142/src/main.ts#L0)):*
<details><summary><i>Previously, tjcouch-sil (TJ Couch) wrote…</i></summary><blockquote>

Looks like code that is unused but has been there a while. I'd prefer not to remove it unless I need to make other changes.
</blockquote></details>

If it's okay, I'd be happy to take a minute to remove it. If it's really unused, I think having it in the code is confusing. In an effort to make it possible for people like me to learn from existing code, it'll be most helpful if we eliminate meaningless cruft.

**Review by tombogle — COMMENTED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/paranext/paranext-extension-text-collection/12)* status: all files reviewed, 1 unresolved discussion (waiting on @tjcouch-sil)

**Comment by tombogle** [comment]
*[`src/main.ts` line 0 at r1](https://reviewable.io/reviews/paranext/paranext-extension-text-collection/12#gh-1367331160:-NhF-Y6k9pxCBdOlE8_Q:b-knfq2z) ([raw file](https://github.com/paranext/paranext-extension-text-collection/blob/466d637f8b719cccd85c01f1adf22608006e7142/src/main.ts#L0)):*
<details><summary><i>Previously, tombogle (Tom Bogle) wrote…</i></summary><blockquote>

If it's okay, I'd be happy to take a minute to remove it. If it's really unused, I think having it in the code is confusing. In an effort to make it possible for people like me to learn from existing code, it'll be most helpful if we eliminate meaningless cruft.
</blockquote></details>

But in the interest of not holding this work up, I'll resolve this

**Review by tombogle — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/paranext/paranext-extension-text-collection/12)* status: :shipit: complete! all files reviewed, all discussions resolved

**Review by tombogle — APPROVED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/paranext/paranext-extension-text-collection/12)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**